acceptance/write_proxy_test.go

```diff
@@ -0,0 +1,78 @@
+package influxtest
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+
+	"github.com/prometheus/common/model"
+)
+
+type Response struct {
+	Status string `json:"status"`
+	Data   struct {
+		ResultType string `json:"resulttype"`
+		Result     []struct {
+			Metric model.Metric     `json:"metric"`
+			Sample model.SamplePair `json:"value"`
+		} `json:"result"`
+	} `json:"data"`
+}
```

Comment:
Instead of defining your own client and handling deserialisation yourself, it might be easier to use the Prometheus client from the official library: https://github.com/prometheus/client_golang/blob/main/api/prometheus/v1/api.go